integration_test/test_compute_metrics.py

```diff
@@ -0,0 +1,89 @@
+from sigopt.xgboost.compute_metrics import (
```

Comment:
I think this file should live under `test/` direction, since it's *not* an integration test.